book_store.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable no-unused-vars */
+const title = document.querySelector('.Book');
+const listB = document.getElementById('book-list');
+let books = [];
+
+const popHtml = () => {
+  listB.innerHTML = '';
+  books = JSON.parse(localStorage.getItem('Libros'));
+
+  if (books === null) {
+    books = [];
+  } else {
+    books.forEach((element) => {
+      listB.innerHTML += `  <li class="book-title">${element.bookTitle}</li>
+      <li class="book-author">${element.author}</li>
+      <button id="btn-remove" onclick="remove()">Remove</button>`;
+    });
+  }
+};
+
+document.addEventListener('DOMContentLoaded', popHtml);
+
+function addBook(e) {
+  e.preventDefault();
+  const bookTitle = (this.querySelector('[name=book]')).value;
+  const author = (this.querySelector('[name=auth]')).value;
+  const book = {
+    bookTitle,
+    author,
+  };
+  this.reset();
+  books.push(book);
+  localStorage.setItem('Libros', JSON.stringify(books));
+  popHtml();
+}
+
+function remove() {
+  const ul = document.getElementById('book-list');
+  ul.parentNode.removeChild(ul);
+}
+
+title.addEventListener('submit', addBook);
```

Comment:
- It would be great to remove items by ID.
(I can't remove books that i added at the list. )
In example you could do something similar like:
``
const removeBook = (bookID) => {
  // bookID can be any other property, e.g. title or author, in as much it is unique
  books = books.filter(({ id }) => bookID !== id); // This assumes that `bookID` parameter is the book id
};
``

Reply:
I checked the code and I'm able to delete items, it looks fixed 😄